src/helpers.c

```diff
@@ -915,23 +915,24 @@ int get_value_from_path(char **contents, const char *path, bool is_abs_path)
 	return ret;
 }
 
-void print_pattern(const char *pattern, int times)
+static void print_char(const char c, int times)
 {
-	while (times > 0) {
-		info("%s", pattern);
-		times--;
-	}
-	info("\n");
+	char *str;
+
+	str = calloc(1, times + 1);
+	memset(str, c, times);
```

Comment:
An option to prevent allocation more memory is to do this:
```printf("%.*s", times, "___________________________________________________________");```
But I prefered to keep it like this.

Reply:
clever! 👏